assets/src/scripts/components/hydrograph/index.js

```diff
@@ -95,8 +95,8 @@ const plotTooltips = function (elem, {xScale, yScale, data}) {
 
     elem.append('rect')
         .attr('class', 'overlay')
-        .attr('width', WIDTH)
-        .attr('height', HEIGHT)
+        .attr('width', layout.width)
+        .attr('height', layout.height)
```

Comment:
I wonder if this could be set to 100% somehow, to avoid the redraw on change?

Reply:
I'll give a try.